src/render/ps2/SDL_render_ps2.c

```diff
@@ -678,9 +678,20 @@ static void
 PS2_DestroyTexture(SDL_Renderer * renderer, SDL_Texture * texture)
 {
     GSTEXTURE *ps2_texture = (GSTEXTURE *) texture->driverdata;
+    PS2_RenderData *data = (PS2_RenderData *)renderer->driverdata;
+
+    if (data == 0)
+        return;
 
+    if(ps2_texture == 0)
```

Comment:
Sorry for the nitpick.
```suggestion
    if (ps2_texture == 0)
```